lib/assessment.rb

```diff
@@ -4,37 +4,38 @@ class Assessment
   # Write ruby code to return the longest entry in the
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
-  def longest(enum)
-    # Your code goes here
+  def longest(enum) { |a, b| a.length <=> b.length }
+    enum.max
```

Comment:
Close, actually! The block needs to be given as an argument to the enumerator's `max` method.
`enum.max { |a, b| a.length <=> b.length }`